unstructured/metrics/evaluate.py

```diff
@@ -208,18 +214,30 @@ def group_text_extraction_accuracy(
             f"Data cannot be aggregated by `{grouping}`."
             f" Check if it's empty or the column is missing/empty."
         )
-    grouped_acc = (
-        df.groupby(grouping)
-        .agg({"cct-accuracy": [_mean, _stdev, "count"]})
-        .rename(columns={"_mean": "mean", "_stdev": "stdev"})
-    )
-    grouped_miss = (
-        df.groupby(grouping)
-        .agg({"cct-%missing": [_mean, _stdev, "count"]})
-        .rename(columns={"_mean": "mean", "_stdev": "stdev"})
-    )
-    grouped_df = _format_grouping_output(grouped_acc, grouped_miss)
-    _write_to_file(export_dir, f"all-{grouping}-agg-cct.tsv", grouped_df)
+    if metric_strategy == "text_extraction":
+        grouped_acc = (
+            df.groupby(grouping)
+            .agg({"cct-accuracy": [_mean, _stdev, _count]})
+            .rename(columns={"_mean": "mean", "_stdev": "stdev", "_count": "count"})
+        )
+        grouped_miss = (
+            df.groupby(grouping)
+            .agg({"cct-%missing": [_mean, _stdev, _count]})
+            .rename(columns={"_mean": "mean", "_stdev": "stdev", "_count": "count"})
+        )
+        grouped_df = _format_grouping_output(grouped_acc, grouped_miss)
+        metric_strategy = "cct"
+    elif metric_strategy == "element_type":
+        grouped_df = (
+            df.groupby(grouping)
+            .agg({"element-type-accuracy": [_mean, _stdev, _count]})
+            .rename(columns={"_mean": "mean", "_stdev": "stdev", "_count": "count"})
+        )
+        grouped_df = _format_grouping_output(grouped_df)
+        metric_strategy = "element-type"
```

Comment:
This looks redundant

Reply:
This just changes `_` to `-` for output file name consistency.